base/docs/helpdb/Dates.jl

```diff
@@ -49,11 +50,55 @@ All characters not listed above are treated as delimiters between date and time
 Dates.DateTime(dt::AbstractString, format::AbstractString)
 
 """
+    format(dt::TimeType, format::AbstractString; locale="english") -> AbstractString
+
+Construct a string by using a `TimeType` and applying the pattern provided by the
```

Comment:
I'll update that. Unfortunately `DateTime <: Dates.TimeType <: Dates.AbstractTime`. I agree the name isn't the best but I won't tackle renaming the types in this PR.